packages/dds/tree/src/test/shared-tree/fuzz/baseModel.ts

```diff
@@ -0,0 +1,45 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import type { DDSFuzzModel, DDSFuzzTestState } from "@fluid-private/test-dds-utils";
+import { validateFuzzTreeConsistency } from "../../utils.js";
+import { fuzzReducer } from "./fuzzEditReducers.js";
+import { SharedTreeFuzzTestFactory, createOnCreate } from "./fuzzUtils.js";
+import type { Operation } from "./operationTypes.js";
+import { takeAsync } from "@fluid-private/stochastic-test-utils";
+import { type EditGeneratorOpWeights, makeOpGenerator } from "./fuzzEditGenerators.js";
+
+export const runsPerBatch = 50;
+// TODO: Enable other types of ops.
+// AB#11436: Currently manually disposing the view when applying the schema op is causing a double dispose issue. Once this issue has been resolved, re-enable schema ops.
+export const editGeneratorOpWeights: Partial<EditGeneratorOpWeights> = {
+	set: 3,
+	clear: 1,
+	insert: 5,
+	remove: 5,
+	intraFieldMove: 5,
+	crossFieldMove: 5,
+	start: 1,
+	commit: 1,
+	abort: 1,
+	fieldSelection: { optional: 1, required: 1, sequence: 3, recurse: 3 },
+	schema: 0,
+	nodeConstraint: 3,
+	fork: 1,
+	merge: 1,
+};
+export const generatorFactory = () => takeAsync(100, makeOpGenerator(editGeneratorOpWeights));
```

Comment:
you probably actually want to export the generator before setting its length, right? gives more flexibility to the caller.

Reply:
actually, i don't think i even need to export the generatorFactory. the goal here is really that we export some model, and that model is also tested at the dds level. all the parts still exist within the package for other compositions.